lib/utils/ipv6-utils.ts

```diff
@@ -0,0 +1,35 @@
+
+/*
+** This policy is required for all the roles which are manging/creating the nodes in the cluster.
+** So, we need it for both cluster node role and karpenter node role.
+** Refer to: https://docs.aws.amazon.com/eks/latest/userguide/cni-iam-role.html#cni-iam-role-create-role
+ */
+import {PolicyDocument} from "aws-cdk-lib/aws-iam";
+
+export function getEKSNodeIpv6PolicyDocument(){
```

Comment:
minor: let's declare the return value in the function signature rather than letting js derive it.